contracts/Controller.sol

```diff
@@ -181,7 +220,7 @@ contract Controller is Initializable, OwnableUpgradeSafe, ReentrancyGuardUpgrade
      * @notice modifier to check if sender is the fullPauser address
      */
     modifier onlyFullPauser {
-        require(msg.sender == fullPauser, "Controller: sender is not fullPauser");
+        require(msg.sender == fullPauser, "CO/err1");
```

Comment:
Can we make the error code all capital? or just `C01`, `C02` .. etc.
The `/err` seems pretty unnecessary.